tests/test_cases/test_cocotb/test_cocotb.py

```diff
@@ -1322,5 +1316,4 @@ def test_bad_attr(dut):
         assert False, "Expected AttributeError"
 
 
-if sys.version_info[:2] >= (3, 5):
-    from test_cocotb_35 import *
+from test_cocotb_35 import *
```

Comment:
There's no need to keep these test files separate any more I guess? Can we combine them into one?

Reply:
Yep, that's a separate task in the tracking issue - so I wasn't intending to do it in this PR.